src/v2/Apps/ArtistSeries/__tests__/ArtistSeriesApp.jest.tsx

```diff
@@ -55,31 +57,34 @@ describe("ArtistSeriesApp", () => {
         const wrapper = await getWrapper()
         expect(wrapper.find("AppContainer").length).toBe(1)
         expect(wrapper.find("ArtistSeriesHeader").length).toBe(1)
+        expect(wrapper.find("ArtistSeriesArtworksFilter").length).toBe(1)
+      })
+
+      describe("ArtistSeriesArtworksFilter", () => {
+        it("renders correctly", async () => {
+          const wrapper = await getWrapper()
+          expect(wrapper.find("ArtworkGrid").length).toBe(2)
+        })
       })
 
       describe("ArtistSeriesHeader", () => {
         describe("desktop", () => {
           describe("with an artist", () => {
             it("renders correctly", async () => {
               const wrapper = await getWrapper()
-              // TODO: expect(wrapper.find("ResponsiveImage").length).toBe(1)
               expect(wrapper.find("ArtistSeriesHeaderLarge").length).toBe(1)
               expect(wrapper.find("ArtistSeriesHeaderSmall").length).toBe(0)
               expect(wrapper.find("ArtistInfo").length).toBe(1)
-              expect(wrapper.find("FollowArtistButton").length).toBe(1)
-              expect(wrapper.find("ArtistSeriesArtworksFilter").length).toBe(1)
-              const html = wrapper.html()
-              expect(html).toContain("Pumpkins")
-              expect(html).toContain("All of the pumpkins")
-              expect(html).toContain("Yayoi Kusama")
-              expect(html).toContain("https://test.artsy.net/yayoi-kusama.jpg")
-              expect(html).toContain("/artist/yayoi-kusama")
-              expect(html).toContain(
-                "/artwork/yayoi-kusama-pumpkin-2222222222222222"
-              )
-              expect(html).toContain(
-                "/artwork/yayoi-kusama-pumpkin-33333333333333333"
+              expect(wrapper.find(HeaderImage).length).toBe(1)
+            })
+
+            it("has a correctly sized header image", async () => {
+              const wrapper = await getWrapper()
+              const expectedUrl = resize(
```

Comment:
I would use the raw output here (and below) instead of calling the function-- that way if the underlying implementation changes we'll catch it here.